repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletResponseWrapper.scala

```diff
@@ -465,6 +498,13 @@ class HttpServletResponseWrapper(originalResponse: HttpServletResponse, headerMo
       originalResponse.flushBuffer()
     }
 
+    if (sentError) {
```

Comment:
`writeBody` actually only writes the bytes from the buffer output stream to the underlying output stream. Technically, I suppose, if the underlying buffer filled up those bytes could be written all the way out to the client (although, not actually, because we buffer at the `PowerFilter` level anyway and Repose would run out of memory before writing those bytes to the client... which is another issue entirely). Otherwise, `sendError` should, by contract, clear the buffer. So calling `sendError` on the underlying response should negate that `commit` you see here.

I figured this was safe to do since if you were dealing with a raw container response, I think you would have the same issue? What happens if you write a bunch to the body of a Jetty response and then call `sendError`?